CIP-2001/README.md

```diff
@@ -0,0 +1,70 @@
+---
+CIP: 2001
+Title: Automated Stake Pool Node Version Reporting
+Authors: Marcus Ubani - hello@larissa.health
+Comments-URI: 
+Type: Standards
+Status: Draft
+Created: 2023-04-09
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP proposes a mechanism to automatically report the node version of stake pools in the Cardano network, 
+making it easier for delegators to identify and delegate to well-maintained pools. 
+This would be achieved by including node version information during KES or node updates, which can then be displayed in wallets.
+The CIP-number is inspired by the novel Odysee in Space (2001, 2010), and seeks to ease the general process for delegators and SPOs. 
+
+Among those members who participated in the discussion in Discord and shared their views and ideas were 
+Tommy Kammerer, George [APEX], DanTup [WEN_K],  Homer [AAA], Vahid [A4G], Rick [RCADA], Sean [ENVY], Rich [ECP]
+
+
+## Motivation
+
+Outdated nodes and unmaintained stake pools pose potential security risks to the Cardano network. 
+By providing an automated way for delegators to identify the node version of stake pools, 
+they can make more informed decisions when delegating their ADA, 
+ultimately contributing to a more secure and better-maintained network.
+
+## Specification
+
+1. During KES or node updates, stake pool operators (SPOs) would submit a new KES update that includes the current node version.
```

Comment:
This seems pretty fragile given that it relies on the honesty of the reporting pool. In general, I think the protocol version reported in made blocks is probably sufficient. You can also query a node's on-chain relays and check if they are supporting the latest network-protocol versions. This is a good way to detect old infrastructure without requiring ledger changes.

Reply:
>You can also query a node's on-chain relays and check if they are supporting the latest network-protocol versions.

I didn't know about this query option. So could this information already be implemented by wallet providers or explorers? 
If the query returns the version, too, then relays could provide an indication of probability for the node version for PBs that haven't produced blocks yet, while block-producing PBs do report anyway.